### ISSUE TITLE
Support Private and Protected keyword

### ISSUE THREAD

**Issue by genius257** [genius257/au3class#8]
For this to work, a new naming for class methods and properties should be made, and it would require a re-work on how IDispatch allows access to the elements. A different lookup method for each Visibility/Accessibility keyword would be needed.

Please note that if the Static keyword is supported first, Static Protected/Private methods and properties is a whole mess that also needs addressing.
